rollbar/lib/transform.py

```diff
@@ -40,3 +43,12 @@ def transform_path(self, o, key=None):
 
     def transform_custom(self, o, key=None):
         return self.default(o, key=key)
+
+    @staticmethod
+    def rollbar_repr(obj: object) -> Optional[str]:
+        r = None
+        if hasattr(obj, '__rollbar_repr__'):
+            r = obj.__rollbar_repr__()
+            if not isinstance(r, str):
+                raise TypeError(...)
```

Comment:
We should probably return an error message like `__repr__` does: `__repr__ returned non-string (type X)`.

Reply:
🤦 Yep that would have been smart. That has been fixed now.